src/app/fyle/merge-expense/merge-expense-2.page.spec.ts

```
@@ -100,7 +96,7 @@ export function TestCases2(getTestBed) {
       formBuilder = TestBed.inject(UntypedFormBuilder);
       expensesService = TestBed.inject(ExpensesService) as jasmine.SpyObj<ExpensesService>;
       component.fg = formBuilder.group({
-        target_txn_id: [, Validators.required],
+        target_expense_id: [, Validators.required],
```

Biome (2.1.2)

[error] This array contains an empty slots..

The presences of empty slots may cause incorrect information and might be a typo.
Unsafe fix: Replace hole with undefined


(lint/suspicious/noSparseArray)